cmake/googletest_download.txt.in

```diff
@@ -10,13 +10,9 @@ project(googletest-download NONE)
 
 include(ExternalProject)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
 ExternalProject_Add(googletest
   GIT_REPOSITORY    https://github.com/google/googletest.git
-  GIT_TAG           release-1.10.0
+  GIT_TAG           release-1.11.0
```

Comment:
Bump google-test version to get a fix for unused variable warning that appears when compiling with gcc 11.